RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy

```diff
@@ -101,7 +106,7 @@ abstract class ExecutionService implements BEExecutionService {
         }
     }
 
-    ExecutionService() {
+    protected ExecutionService() {
```

Comment:
The constructor is not used except in the initialize method above. This is a singleton and the proper way to initialize it seems to call the initialize method, not the constructor directly.